<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>日期时间数据</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>日期时间</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnGetTime">
        <property name="text">
         <string>读取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>字符串显示</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit"/>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLineEdit" name="editTime">
        <property name="inputMask">
         <string>99:99:99</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnDebugTime">
        <property name="text">
         <string>qDebug--Time</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QPushButton" name="btnSetTime">
        <property name="text">
         <string>设置时间fromString</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="editDate">
        <property name="inputMask">
         <string>9999-99-99</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnDebugDate">
        <property name="text">
         <string>qDebug--Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2" colspan="2">
       <widget class="QPushButton" name="btnSetDate">
        <property name="text">
         <string>设置日期fromString</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>日期时间</string>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QLineEdit" name="editDateTime">
        <property name="inputMask">
         <string>9999-99-99 99:99:99</string>
        </property>
        <property name="text">
         <string>-- ::</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnDebugDateTime">
        <property name="text">
         <string>qDebug--DateTime</string>
        </property>
       </widget>
      </item>
      <item row="6" column="2" colspan="2">
       <widget class="QPushButton" name="btnSetDateTime">
        <property name="text">
         <string>日期时间fromString</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>日历选择</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>选择的日期：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editCalendar"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
